### DISCUSSION TITLE
Custom env by PR

### DISCUSSION THREAD

**Discussion by jonyw4** [vercel/vercel#4608] [Help] [answered]
Hi guys,
I started a project that use a monorepo with this structure:
- Webapp (NextJS in Vercel)
- Server ([Vendure ](vendure.io)in Heroku)

In every pull request should create a new [Review app](https://devcenter.heroku.com/articles/github-integration-review-apps) in heroku and a preview in vercel, but i have a problem. I need to a env var of nextjs app point to the server deployed by the heroku.

Heroku review app use the  `Identifier + PR number` url pattern.

In [Vercel docs](https://vercel.com/docs/v2/build-step#system-environment-variables) i didn't found a env based on PR number, so i am stucked in this part...

How i can tell to my webapp app what URL will be the heroku app?

**Comment by jonyw4** [comment, marked as answer]
I resove this issue by using a github action of vercel cli by passing build-env:
https://github.com/amondnet/vercel-action/issues/13#issuecomment-641452741

**Reply by jacobdo2** [reply]
amondnet does not seem to support passing multiple environment variables, at least not in the way documented in Vercel CLI documentation. Did you manage to pass multiple env variables?
